.github/workflows/cd.yml

```diff
@@ -1,11 +1,14 @@
-# cspell:ignore noreply pypa sdist
+# cspell:ignore bdist noreply pypa sdist
 
 name: CD
 
 on:
   create:
     tags:
       - "[0-9]+.[0-9]+.[0-9]+*"
+  release:
+    types:
+      - created
```

Comment:
why do we still need the `create: tags: ...` part?

Reply:
Actually I prefer the tags trigger, as that allows filtering on semantic versions. But tbh I don't understand why the existing tag trigger does not work anymore. That's why I added a release trigger as well.

Reply:
Hmm ok